### PR TITLE
webui: Display node architecture along with the CPU info

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

File: crowbar_framework/app/models/node_object.rb [hunk 1]
```diff
@@ -553,10 +553,26 @@ def memory
...
   end
 
+  def cpu_arch
+    if !cpu.blank? && !architecture.blank?
+      "#{cpu} (#{architecture})"
+    elsif !cpu.blank?
+      cpu
+    elsif !architecture.blank?
+      architecture
+    else
```
Comment (tboerger) [inline crowbar_framework/app/models/node_object.rb]: This is indeed redundant :)

**Comment by tboerger** [comment]
Drop the `else` than I'm fine.

**Comment by vuntz** [comment]
@tboerger done

File: crowbar_framework/app/models/node_object.rb [hunk 2]
```diff
@@ -553,10 +553,26 @@ def memory
     @node["memory"]["total"] rescue nil
   end
 
+  def architecture
+    @node["kernel"]["machine"] rescue nil
```
Comment (tboerger) [inline crowbar_framework/app/models/node_object.rb]: Let's ignore that for now.

**Comment by tboerger** [comment]
:+1:

**Pull request merged**